tests/tests-misc.php

```diff
@@ -477,4 +477,22 @@ public function test_edd_is_host() {
 		$this->assertTrue( edd_is_host( 'WP Engine' ) );
 		$this->assertTrue( edd_is_host( 'WPEngine' ) );
 	}
+
+	public function test_edd_update_option(){
+		$key   = 'some-setting';
+		$value = 'some-value';
+		$isset = edd_get_option( $key, false );
+
+		// The option shouldn't exist
+		$this->assertFalse( $isset );
+
+		$updated = edd_update_option( $key, $value );
+
+		// The option should have succesfully updated
+		$this->assertTrue( $updated );
+
+		// The option retrieve should be equal to the one we set
+		$this->assertEquals( $value, edd_get_option( $key, false ) );
+
```

Comment:
Let's add a test for removing an option, like this: `edd_update_option( 'test_mode', false );`